master/static/migrations/20240125145347_index-some-alloc-tasks-fields.tx.up.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX ix_allocations_task_id ON allocations USING btree (task_id);
```

Comment:
Nit (non-blocking): `btree` is the default index method, so you can just do `CREATE INDEX ix_allocations_task_id ON allocations (task_id)`

Reply:
I guess there is a very slight argument for keeping the index type explicit? 

It seems we do both in the codebase a bunch